catroid/src/main/java/org/catrobat/catroid/content/actions/ArduinoSendDigitalValueAction.java

```diff
@@ -66,12 +66,7 @@ protected void begin() {
 					interpretationException);
 		}
 
-		if ((!restart)
-				&& (pinNumberInterpretation == 2)
-				|| (pinNumberInterpretation == 4)
-				|| (pinNumberInterpretation == 8)
-				|| (pinNumberInterpretation == 12)
-				|| (pinNumberInterpretation == 13)) {
+		if (!restart) {
 			this.pin = pinNumberInterpretation;
 			this.value = pinValueInterpretation;
 		}
```

Comment:
Shall I branch [#1953](https://github.com/Catrobat/Catroid/pull/1953) off of this pull request's branch to avoid the merge conflict?
see your previous suggestion on a similar situation:
>You could branch of from this PR, add a fix commit and open a PR now. Then the changes are already reviewable. When the other PRs are merged, I'll drop a comment on the new PR, so you just have to rebase the branch or, if you make changes in this PR, cherry-pick the commit. Overall, it'll be simple and smooth.
Additionally I could then include the refactoring in [#1953](https://github.com/Catrobat/Catroid/pull/1953) - or is that a bad idea?